src/atom-environment.coffee

```diff
@@ -1092,6 +1100,14 @@ class AtomEnvironment extends Model
   dispatchContextMenuCommand: (command, args...) ->
     @commands.dispatch(@contextMenu.activeElement, command, args)
 
+  dispatchUrlMessage: (uri) ->
+    if @packages.hasLoadedInitialPackages()
```

Comment:
Not saying you should do it as part of this PR, but it would be cool to have a `getInitialPackagesLoadPromise` method.